playbooks/intel/bmra_features.yml

```diff
@@ -43,4 +43,7 @@
     - role: bond-cni-install
       when: bond_cni_enabled | default(true)
       tags: bond-cni
+    - role: sst-bf-configure
```

Comment:
please move to the kube-node configuration in the `infra` playbook, SST-BF must be enabled before NFD and CMK installation